src/database/DatabaseManager.js

```diff
@@ -58,13 +58,36 @@ export class DataBaseManager {
     async getAdminId(username){
         let id = null;
         logger.debug(`Getting the id of the user ${username}`);
-        await Administrator.findOne({login:username},function(err,username) {
+        let user = await Administrator.findOne({login:username},function(err,username) {
             if (err || username == null) logger.debug(`Impossible to find username`);
-            else id = username._id;
+            else{
+                logger.debug(`User found: ${username}`);
+            }
```

Comment:
Essaye d'être constant sur la structure des blocs if comme ça :
```
if (...) 
  instruction
else 
  instruction
```
ou
```
if (...) {
  instruction
} else { 
  instruction
}
```
Mais pas un mix des deux.

Reply:
Corrected in the new commit. Changed the typo for all the if structures that were incorrect in the file.